documentation/modules/configuring/proc-config-mirrormaker2-replication.adoc

```diff
@@ -178,9 +180,9 @@ spec:
             name: aws-creds
             key: awsSecretAccessKey
 ----
-<1> The Kafka Connect xref:type-KafkaConnectSpec-reference[version].
+<1> The Kafka Connect xref:type-KafkaConnectSpec-reference[version] used with the MirrorMaker 2.0 deployment, which must be from the same Kafka version as MirrorMaker 2.0.
```

Comment:
I think this should say something like _Defines the version of Kafka Connect and Mirror Maker 2 which will be used. They will be always the same_.

Reply:
Updated to:

_The Kafka Connect and Mirror Maker 2.0 xref:type-KafkaConnectSpec-reference[version], which will always be the same._